docs/nodes/nodes-library/trigger-nodes/TypeformTrigger/README.md

```diff
@@ -24,9 +27,77 @@ The final workflow should look like the following image.
 
 ### 1. Typeform Trigger node
 
+This node will trigger the workflow when a form response is submitted.
```

Comment:
You should add a link to the Typeform along with the datatype for the questions. If it is different for the users, this example workflow wouldn't work for them. I'd recommend having two questions top: Name and Email.